nixos/modules/flyingcircus/roles/loghost.nix

```diff
@@ -69,6 +69,35 @@ let
         "echo -n $text | sha256sum | cut -f1 -d \" \" > $out")
       );
 
+  logstashSSLHelper = pkgs.writeScriptBin "logstash_ssl_import" ''
+    #!${pkgs.bash}/bin/bash
+    set -e
+
```

Comment:
Insert a comment that this script is meant to be used interactively by an administrator and that permissions on the puppet master have to be granted manually.